packages/encryption/tests/messageSignature.test.ts

```diff
@@ -0,0 +1,19 @@
+import { decodeMessage, encodeMessage, hashMessage } from '../src/messageSignature';
+
+test('encodeMessage / decodeMessage', () => {
```

Comment:
can we add a test w/ a manually constructed "correct" encoded message?

Reply:
@janniks I added 3 encoded messages, let me know if it's explicit enough or if I am missing something.